src/core/olaf.ts

```diff
@@ -0,0 +1,70 @@
+import * as TelegramBot from 'node-telegram-bot-api';
+import { Message } from 'node-telegram-bot-api';
+
+import preferencesDashboard from '../dashboard/preferences-dashboard';
+import MessageSender from './messageSender';
+import MessageHandler from './messageHandler';
+import MessageRouter from './messageRouter';
+import EndUseCaseResponse from '../classes/EndUseCaseResponse';
+import UseCase from '../interfaces/useCase';
```

Comment:
Controller, coordinator oder du packst das in main.ts, aber olaf.ts ?

Reply:
Wir können das auch OlafApplication oder OlafPDA nennen, aber ich finde Controller/Coordinator zu unspezifisch.